userguide/system.rst

```diff
@@ -2523,25 +2527,26 @@ and described in
    | Setting        | Value          | Description                                                                                                                                           |
    |                |                |                                                                                                                                                       |
    +================+================+=======================================================================================================================================================+
-   | Disabled       | checkbox       | when checked, administratively disable failover which changes the :guilabel:`HA Enabled` icon to :guilabel:`HA Disabled` and                          |
-   |                |                | activates the :guilabel:`Master` field; an error message is generated if the standby node is not responding or failover has not been                  |
-   |                |                | configured                                                                                                                                            |
+   | Disabled       | checkbox       | When enabled, administratively disable failover which changes the :guilabel:`HA Enabled` icon to :guilabel:`HA Disabled` and                          |
```

Comment:
Break up this sentence.

Disable failover. The :guilabel:`HA Enabled` icon changes to :guilabel:`HA Disabled` and